internal/authtoken/authtoken.go

```diff
@@ -92,7 +92,7 @@ func newAuthTokenId() (string, error) {
 	const op = "authtoken.newAuthTokenId"
 	id, err := db.NewPublicId(AuthTokenPrefix)
 	if err != nil {
-		return "", errors.Wrap(err, op, errors.WithCode(errors.Io))
+		return id, errors.Wrap(err, op)
```

Comment:
oh no sorry, I meant you should return on line 97 `id, nil` to make it explicit instead of returning the `err` which h will anyway be nil, this should be `"", errors.Wrap...` as you had it